apps/server/src/modules/files-storage/helper/scan-status.spec.ts

```diff
@@ -1,5 +1,5 @@
 import { fileRecordFactory } from '@shared/testing';
-import { deriveStatusFromSource, isStatusBlocked } from '.';
+import { deriveStatusFromSource } from '.';
 import { ScanStatus } from '../entity';
 
 describe('Scan Status Helper', () => {
```

Comment:
deriveStatusFromSource-Test fehlt eigentlich noch der Fall für ScanStatus.Error. Der hätte eigentlich schon in meinem Branch ergänzt werden müssen.